merkle/log_proofs_test.go

```diff
@@ -151,7 +151,7 @@ func TestCalcConsistencyProofNodeAddresses(t *testing.T) {
 	// Expected consistency proofs built from the examples in RFC 6962. Again, in
 	// our implementation node layers are filled from the bottom upwards.
 	//
-	//                hash6                         hash7
+	//                hash5                         hash7
```

Comment:
It's not wrong but it probably invalidates the comment about the examples being taken from the RFC.

Reply:
True. In fact, this tests has/will have more tests than in RFC, so I'll just remove the reference to RFC.

Reply:
Done.